spec/requests/universal_links_spec.rb

```diff
@@ -28,7 +28,22 @@
         json_response = JSON.parse(response.body)
         expect(response).to have_http_status(:ok)
 
-        expect(json_response.dig("applinks", "apps")).to match_array([forem_app_id])
+        expect(json_response.dig("applinks", "apps")).to match_array([])
+        json_response.dig("applinks", "details").each do |hash|
+          expect(hash["appID"]).to eq(forem_app_id)
+          expect(hash["paths"]).to match_array(["/*"])
+        end
+      end
+    end
+
+    context "when non-public Forem instance" do
+      it "responds with applinks support for Forem app" do
```

Comment:
We are trying to get a bit more consistent about using `aggregate_failures` for specs with more than one assertion:

```suggestion
      it "responds with applinks support for Forem app", :aggregate_failures do
```